src/StandaloneWriterPluginManager.php

```diff
@@ -13,6 +13,7 @@
 
 class StandaloneWriterPluginManager implements ContainerInterface
 {
+    /** @var string[] */
```

Comment:
We should be a bit more precise about the keys here

Reply:
done